src/v1/providers/firestore.ts

```diff
@@ -31,11 +31,8 @@ import * as logger from "../../logger";
 import { CloudFunction, Event, EventContext, makeCloudFunction } from "../cloud-functions";
 import { DeploymentOptions } from "../function-configuration";
 
-/** @hidden */
 export const provider = "google.firestore";
```

Comment:
Maybe this is a personal nit but, shouldn't these three const strings be uppercased?

Reply:
Please mark this @internal instead.

if you mean variable name should be caps, maybe! if you mean the string value, i don't think so (they might map to eventflow event type)

Reply:
renamed to caps and added `@internal`